src/MoBi.Presentation/Presenter/EditParametersInContainerPresenter.cs

```diff
@@ -175,13 +179,16 @@ private void createParameterCache(IReadOnlyList<IParameter> parametersToEdit, IR
          releaseParameters();
          _allParametersDTO.Clear();
          _allParametersDTO.AddRange(_parameters.MapAllUsing(_parameterToDTOParameterMapper));
-         
+
          _allParametersDTO.AddRange(parametersForIndividualParameters(individualParameters, selectedIndividual));
       }
 
       private IReadOnlyList<ParameterDTO> parametersForIndividualParameters(IReadOnlyList<IndividualParameter> individualParameters, IndividualBuildingBlock selectedIndividual)
       {
-         return individualParameters.Select(individualParameter => _individualParameterToDTOParameterMapper.MapFrom(selectedIndividual, individualParameter)).ToList();
+         _individualParameterCache.Clear();
```

Comment:
this is also not expected. You are returnign a list from something and it has side effects. It's easier to understand when method with side effects return void. 
Maybge it's a createIndividualParameterCache method and then you use the cache after or sthg.